openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/ConfigureIngestion.tsx

```diff
@@ -436,6 +438,25 @@ const ConfigureIngestion = ({
             {getPipelineFieldToggles()}
           </Fragment>
         );
+
+      case ServiceCategory.ML_MODAL_SERVICES:
```

Comment:
nit - I've seen in the code the ML Models being referred both as `ML Model` and `ML Modal`. It should be `Model`. Leaving it here in case we want to clean the naming in the future